nvm.psm1

```diff
@@ -371,7 +371,18 @@ function Install-NodeVersion {
                 throw $errMsg
             }
 
-            Move-Item (Join-Path (Join-Path $unpackPath 'nodejs') '*') -Destination $versionPath -Force
+            # build expected path to nodejs
+            $expectedPath = Join-Path $unpackPath 'nodejs'
+
+            if (-Not (Test-Path $expectedPath)) {
+                $expectedPath = Join-Path (Join-Path $unpackPath 'PFiles64') 'nodejs'
```

Comment:
We need to check if the folder is `PFiles` or `PFiles64`, depending on the architecture of the install (64bit vs 32bit).

Can you expand this check to factor that in.

Reply:
Done